flterm.c

```diff
@@ -610,8 +755,16 @@ static void do_terminal(
 								cmdline, cmdline_address,
 								initrd_image, initrd_address);
 						}
-					} else {
+					} else if(c != 'C') {
+						/* If XMODEM start not detected, continue looking
+						for SFL start. */
```

Comment:
Indentation...

Reply:
Could you please elaborate on what's wrong with the indentation?
 https://github.com/cr1901/flterm/blob/master/flterm.c#L747-L769

Reply:
See https://github.com/cr1901/flterm/blob/9e11d2c7e32cc196b5f7aa6109c9f817d43c91cf/flterm.c#L467-L470 for example.

This style of comment should be wrapped like this;
```c
/* If XMODEM start not detected, continue looking
 * for SFL start.
 */
```